test/components/numericAxisTests.ts

```diff
@@ -400,4 +400,26 @@ describe("NumericAxis", () => {
     });
     svg.remove();
   });
+
+  it("renders tick labels properly when the domain is reversed", () => {
+    var SVG_WIDTH = 300;
+    var SVG_HEIGHT = 100;
+    var svg = generateSVG(SVG_WIDTH, SVG_HEIGHT);
+    var scale = new Plottable.Scale.Linear();
+    scale.domain([3, 0]);
+    var baseAxis = new Plottable.Axis.Numeric(scale, "bottom");
+    baseAxis.renderTo(svg);
+    var tickLabels = (<any> baseAxis)._element.selectAll(".tick-label")
+      .filter(function(d: any, i: number) {
+        var visibility = d3.select(this).style("visibility");
+        return (visibility === "visible") || (visibility === "inherit");
+      });
```

Comment:
Can we space this a bit so that it's a bit more clear what's going on (shift the filter call more to the right)